pkg/machinery/kernel/kernel.go

```diff
@@ -31,6 +31,14 @@ var DefaultArgs = []string{
 	"ima_template=ima-ng",
 	"ima_appraise=fix",
 	"ima_hash=sha512",
+	// TODO: configure it properly
+	"talos.selinux=permissive",
```

Comment:
audit can probably use these libraries:

* https://github.com/elastic/go-libaudit
* https://github.com/slackhq/go-audit

Reply:
we can probably use this https://github.com/elastic/go-libaudit/blob/main/audit.go#L119 by passing in netlink from https://pkg.go.dev/github.com/mdlayher/netlink#example-Conn-ListenMulticast

Reply:
Even without audit options I get audit logs in dmesg. Should I explicitly disable audit for now then, until I redirect it?

Reply:
```
audit=          [KNL] Enable the audit sub-system
                        Format: { "0" | "1" | "off" | "on" }
                        0 | off - kernel audit is disabled and can not be
                            enabled until the next reboot
                        unset - kernel audit is initialized but disabled and
                            will be fully enabled by the userspace auditd.
                        1 | on - kernel audit is initialized and partially
                            enabled, storing at most audit_backlog_limit
                            messages in RAM until it is fully enabled by the
                            userspace auditd.
                        Default: unset
```

I think dmesg will always log that, this is probably to control enabling the netlink socket maybe?

Reply:
fixed by auditd PR